simp_le.py

```diff
@@ -1511,11 +1511,14 @@ def test_it(self):
         files = ('account_key.json', 'key.pem', 'full.pem')
         with self._new_swd():
             self.assertEqual(EXIT_RENEWAL, self._run(args))
+            # Set time to the past to ensure that changes are detected.
```

Comment:
It is required to avoid false negatives when the test fails as the mtime has only second-granularity. Alternative would be `time.sleep(2)`, but that is even uglier. Or are you thinking about using a st_mtime relative value?